src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -119,53 +119,57 @@ var editCommentHandler = function(e) {
 
 var deleteCommentHandler = function(e) {
     var submitButton = $(this);
-    var formObject = $(this).parent();
-    var deletedCommentRow = $(this).closest('li');
-    var formData = formObject.serialize();
-    var editForm = submitButton.parent().next().next().next();
-    var frCommentList = submitButton.closest('.comments');
-    var panelHeading = $(this).parents("[id^='panel_display-']").find('.panel-heading').first();
-    
-    e.preventDefault();
-    
-    $.ajax({
-        type: 'POST',
-        url: submitButton.attr('href') + '?' + formData,
-        beforeSend: function() {
-            submitButton.html("<img src='/images/ajax-loader.gif'/>");
-        },
-        error: function() {
-            if (editForm.is(':visible')) {
-                setFormErrorMessage(editForm.find('div > a'), 'Failed to delete comment. Please try again.');
-            } else if (frCommentList.parent().find('div.delete_error_msg').length === 0) {
-                frCommentList.after('<div class="delete_error_msg alert alert-danger">'
-                                    + 'Failed to delete comment. Please try again.</div>');
-            }
-            submitButton.html('<span class="glyphicon glyphicon-trash glyphicon-primary"></span>');
-        },
-        success: function(data) {
-            if (data.isError) {
+
+    deleteResponseComment(function() {
```

Comment:
Is it possible to refactor the code somehow?

It is not immediately clear why a function named "deleteResponseComment()" accepts a function as its parameters. In fact, the "deleteResponseComment()" function contains only the code relating to the modal.

Also, now that the ajax code is pushed one level deeper, perhaps we can start extracting smaller functions so it is more readable.